src/Services/testimonialsService.js

```diff
@@ -1,6 +1,10 @@
 import axios from 'axios';
 
-const TESTIMONIAL_URL = 'http://ongapi.alkemy.org/api/testimonials/';
+const GET_TESTIMONIAL_URL = process.env.REACT_APP_GET_TESTIMONIAL;
+const GET_ALL_TESTIMONIALS_URL = process.env.REACT_APP_GET_ALL_TESTIMONIALS;
+const POST_TESTIMONIAL_URL = process.env.REACT_APP_POST_TESTIMONIAL;
+const UPDATE_TESTIMONIAL_URL = process.env.REACT_APP_UPDATE_TESTIMONIAL;
+const DELETE_TESTIMONIAL_URL = process.env.REACT_APP_DELETE_TESTIMONIAL;
```

Comment:
if the url are the same for all the endpoints instead of create one for each create only one

Reply:
also, add the missing endpoints in the service